File: ConsoleAppVb/Methods.vb
Public Class Methods

    ' Tuple
    Public Shared Function GetFullName() As (String, String)
        Console.Write("What is your first name: ")
        Dim firstName As String = Console.ReadLine()

        Console.Write("What is your last name: ")
        Dim lastName As String = Console.ReadLine()

        Return (firstName, lastName)
    End Function

End Class

File: ConsoleAppCs/Methods.cs

namespace ConsoleAppCs
{
    public static class Methods
    {

        // Tuple
        public static (string firstName, string lastName) GetFullName()
        {
            Console.Write("What is your first name: ");
            string firstName = Console.ReadLine();

            Console.Write("What is your last name: ");
            string lastName = Console.ReadLine();

            return (firstName, lastName);
        }

    }
}
